android/app/src/main/java/org/fossasia/openevent/activities/SessionDetailActivity.java

```diff
@@ -220,6 +221,27 @@ protected int getLayoutResource() {
     }
 
     @Override
+    public void onBackPressed() {
+        if(fabSessionBookmark.getVisibility() == View.GONE){
+            /** hide fragment again on back pressed and show session views **/
+            FrameLayout mapFragment = (FrameLayout) findViewById(R.id.content_frame_session);
```

Comment:
Hey, thanks for the PR.
We are already using Butterknife in the project so it would be better if you use it instead. 🙂